molecule/commands.py

```diff
@@ -442,26 +446,32 @@ class Test(AbstractCommand):
     Runs a series of commands (defined in config) against instances for a full test/verify run.
 
     Usage:
-        test [--platform=<platform>] [--provider=<provider>] [--destroy=<destroy>] [--debug]
+        test [--platform=<platform>] [--provider=<provider>] [--destroy=<destroy>] [--debug] [--sudo]
 
     Options:
         --platform=<platform>  specify a platform
         --provider=<provider>  specify a provider
         --destroy=<destroy>    destroy behavior (passing, always, never)
         --debug                get more detail
+        --sudo                 run tests with sudo
     """
 
     def execute(self):
         if self.static:
             self.disabled('test')
 
-        command_args, args = utilities.remove_args(self.command_args,
-                                                   self.args, ['--destroy'])
+        command_args, args = utilities.remove_args(
+            self.command_args, self.args, self.command_args)
```

Comment:
Just to point this out - this performs the same thing as before. This just makes it easier to add in flags that dont match the commands in the test chain.